### PR TITLE
issue2pr password

### PR DESCRIPTION
**Pull request by Culprit-CZ**
Password with special chars

http://phpbbsocialnetwork.com/viewtopic.php?f=2&t=4227

### CODE REVIEW

**Comment by Kamahl19** [comment]
MA

File: root/socialnet/js/m.profile.js [hunk 1]
```diff
@@ -74,7 +74,7 @@
 							        templates : {
 							            wrapper : _.template('<div class="sn-us-mentions-input-box"></div>'),
 							            autocompleteList : _.template('<div class="sn-us-mentions-autocomplete-list"></div>'),
-							            mentionsOverlay : _.template('<div class="sn-us-mentions"><div></div></div>'),
+							            mentionsOverlay : _.template('<div class="sn-us-mentions"><div></div></div>')
```
Comment (senky) [inline root/socialnet/js/m.profile.js:77]: this change should be removed, I think...

File: root/socialnet/js/m.userstatus.js [hunk 2]
```diff
@@ -504,7 +504,7 @@
 			        templates : {
 			            wrapper : _.template('<div class="sn-us-mentions-input-box"></div>'),
 			            autocompleteList : _.template('<div class="sn-us-mentions-autocomplete-list"></div>'),
-			            mentionsOverlay : _.template('<div class="sn-us-mentions"><div></div></div>'),
+			            mentionsOverlay : _.template('<div class="sn-us-mentions"><div></div></div>')
```
Comment (senky) [inline root/socialnet/js/m.userstatus.js:507]: this one too

**Comment by Culprit-CZ** [comment]
Canceled

**Pull request closed without merging**